ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -963,6 +963,8 @@ private class PostgresLedgerDao(
     dbDispatcher.executeSql { implicit conn =>
       SQL_SELECT_PARTIES
         .as(PartyDataParser.*)
+        // TODO: isLocal should be based on equality of participantId reported in an
```

Comment:
Please open a github issue for this and reference the issue here in a code comment.

Reply:
Issue #2026